server/bound_queries.cc

```diff
@@ -1449,6 +1449,13 @@ execute(RowProcessor processor,
     typedef std::map<RowKey, GroupMapValue> GroupByMapType;
     std::vector<GroupByMapType> accum(numBuckets);
 
+    for (const auto & c: select.clauses) {
+        if (typeid(*c.get()) == typeid(WildcardExpression)) {
```

Comment:
I would implement an isWildcardExpression() function in SqlExpression, with a default implementation that returns false, and override this for WildcardExpression.  It reduces coupling and will allow (for example) a derivative of WildcardExpression to work properly.  Direct typeid checks like that should usually be replaced with a virtual method.